### DISCUSSION TITLE
Modifying CORE-GUI to Run in Execution Mode with .XML Input in the Latest Version

### DISCUSSION THREAD

**Discussion by faceb912** [coreemu/core#832] [Q&A] [answered]
Hi all, 
In earlier versions of CORE, I could initiate the core-gui in execution mode by running the command core-gui -start followed by the path to an .imn file. However, in the latest version, it appears that this argument has been removed. To reintroduce this functionality, I'd like to know which script or configuration file I need to modify and what changes are necessary to enable the core-gui to run in execution mode with a .XML file as input. Can you provide guidance on the specific file or script I should modify and the changes needed?

**Comment by bharnden** [comment, marked as answer]
The aspect of loading an XML file to the running daemon can be done from the `core-cli` executable.

```shell
usage: core-cli xml [-h] -f FILE [-s]

optional arguments:
  -h, --help            show this help message and exit
  -f FILE, --file FILE  xml file to open (default: None)
  -s, --start           start the session? (default: False)
```

**Reply by faceb912** [reply]
@bharnden Thank you for your prompt response! Your answer was helpful, and I appreciate your assistance.
